docs/source/model-registry.rst

```diff
@@ -230,6 +230,21 @@ token = <your Databricks shard access token>
     mlflow.set_registry_uri("databricks-uc://my-databricks-shard1")
 
 
+OSS Unity Catalog Model Registry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To use an `OSS Unity Catalog <https://www.unitycatalog.io/>` server as your MLflow model registry, specify the UC server address in the MLflow registry URI in the format ``"uc:http://localhost:8080"``.  If your unity catalog server is configured to use authentication, set the environmental variable ``"MLFLOW_UC_OSS_TOKEN"``.
+
+**Use the OSS unity catalog model registry using a bearer token**
+
+.. code-block:: python
+
+    import mlflow
+    import os
+
+    mlflow.set_registry_uri("uc:http://localhost:8080")
+    os.environ["MLFLOW_UC_OSS_TOKEN"] = "<your OSS UC access token>"
```

Comment:
Hardcoding a token is not a good practice. Can we use `getpass`?

Reply:
We can but I'm worried about making the cutoff for Friday as I have very little time left to be adding new code/testing before then.  We can add that as an improvement for after release.  UC OSS is still v0.2 right now and is hardly what I would consider production ready as it is.

Reply:
Do you want that note added here @BenWilson2 ?

Reply:
We don't need that full note, just a comment in the code block to remind users to set the env var. We remind people in other places in the docs by doing an assert on the existence of the key via something like:
```python
assert "MLFLOW_UC_OSS_TOKEN" in os.environ.keys()
```

Reply:
@kriscon-db 

We just need to add this.

```python
from getpass import getpass

os.environ["MLFLOW_UC_OSS_TOKEN"] = getpass(...)
```

We can improve this as a follow-up.